rotkehlchen/api/v1/resources.py

```diff
@@ -1088,13 +1088,29 @@ class DataImportResource(BaseResource):
     upload_schema = DataImportSchema()
 
     @use_kwargs(upload_schema, location='json')  # type: ignore
-    def put(self, source: Literal['cointracking.info', 'cryptocom'], file: Path) -> Response:
+    def put(
+        self,
+        source: Literal[
+            'cointracking.info',
+            'cryptocom',
+            'blockfi-transactions',
+            'blockfi-trades',
+            'nexo',
+        ],
+        file: Path,
+    ) -> Response:
         return self.rest_api.import_data(source=source, filepath=file)
 
     @use_kwargs(upload_schema, location='form_and_file')  # type: ignore
     def post(
             self,
-            source: Literal['cointracking.info', 'cryptocom'],
+            source: Literal[
```

Comment:
you keep repeating this Literal. Better define it as a type somewhere and import it